src/mame/audio/8080bw.cpp

```diff
@@ -1397,3 +1404,276 @@ WRITE8_MEMBER( _8080bw_state::darthvdr_08_w )
+
+	/************************************************/
+	/* From 555 				                    */
+	/************************************************/
+	/* TODO: find real noise freq and amplitude */
+	/* width was simulated with ltspice using Claybuster schematic as a source and it's value is about 51ms */
+	DISCRETE_NOISE(CANE_76477_PIN6,
+				1,                  	    /* ENAB */
+				1280,                       /* FREQ - Guessed */
+				1,		                    /* AMP  */
+				0)                          /* BIAS - fake AC is fine*/
+	DISCRETE_CLAMP(CANE_555_CLAMPED,
+				CANE_76477_PIN6,			/* input node */
+				0.0,				 		/* minimum */
+				5.0)						/* maximum */
+	DISCRETE_ONESHOT(CANE_555_ONESHOT,
+					 CANE_555_EN,			/* trigger node */
+					 1,						/* amplitude node or static value */
+					 0.05,					/* width (in seconds) node or static value - 50 ms*/
+					 DISC_ONESHOT_FEDGE | DISC_ONESHOT_RETRIG)	/* type of oneshot static value */
+
+	DISCRETE_MULTIPLY(CANE_TMP_SND, CANE_555_CLAMPED, CANE_555_ONESHOT)
+	DISCRETE_MULTIPLY(CANE_SFX_SND, CANE_TMP_SND, CANE_VR3)
+
+/*****************************************************************************
+*
+* Music Generator (TOS)
+*
+* Values for this section of the sound hardware where derived from comments
+* in the source code and the analysis of TOS.ED sources.
+* 
+* For further info look at the relevant comments reported into 
+* drivers/8080bw.cpp
+*
+******************************************************************************/
+	DISCRETE_NOTE(CANE_MUSIC_NOTE_PF, 1, CANE_H64, CANE_MUSIC_DATA, 255, 1, DISC_CLK_IS_FREQ)
+	DISCRETE_CRFILTER(CANE_MUSIC_NOTE, CANE_MUSIC_NOTE_PF, RES_K(10), CAP_U(0.1))  // high pass filter
+	DISCRETE_MULTIPLY(CANE_MUSIC_SND, CANE_MUSIC_NOTE, CANE_VR2)
+
+/******************************************************************************
+*
+* From 76477 output
+*
+******************************************************************************/
+	DISCRETE_MULTIPLY(CANE_EXP_SND, CANE_EXP_STREAM, CANE_VR1)	
+
+/******************************************************************************
+*
+* Final Mixing and Output
+*
+******************************************************************************/
+	DISCRETE_ADDER3(CANE_SOUND_OUT, CANE_SND_EN, CANE_SFX_SND, CANE_EXP_SND, CANE_MUSIC_SND)
+	DISCRETE_OUTPUT(CANE_SOUND_OUT, 1)
+
+//LOG
+/*
+	DISCRETE_WAVLOG1(CANE_EXP_STREAM, 1)
+	DISCRETE_WAVLOG1(CANE_EXP_SND, 1)
+	DISCRETE_WAVLOG1(CANE_TMP_SND, 1)
+	DISCRETE_WAVLOG1(CANE_SFX_SND, 1)
+	DISCRETE_WAVLOG1(CANE_MUSIC_NOTE, 1)
+	DISCRETE_WAVLOG1(CANE_MUSIC_SND, 1)
+	DISCRETE_WAVLOG1(CANE_SOUND_OUT, 1)
+*/
+DISCRETE_SOUND_END
+
+TIMER_DEVICE_CALLBACK_MEMBER(cane_audio_device::cane_vco_voltage_timer)
+{
+	double voltage;
+	voltage = 5 * (1 - exp(- (m_cane_vco_timer->fire_time().as_double() - m_cane_vco_rc_chargetime) / 47));
+
+	logerror("t = %d\n", m_cane_vco_timer->fire_time().as_double() - m_cane_vco_rc_chargetime);
+	logerror("vco_voltage = %d\n", voltage);
```

Comment:
These should use `LOG` from `logmacro.h` so they don't spam unless `VERBOSE=1` is defined.

Reply:
Ok